src/components/Card/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement } from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import Feather from '@expo/vector-icons/Feather';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Container, Photo, Name, Budget, ContentContainer } from './styles';
+
+interface CardProps {
+  name: string;
+  image: string;
+  budget: string;
+}
+
+export function Card({ name, image, budget }: CardProps): ReactElement {
+  const navigation = useNavigation();
+
+  function handleNavigateToProject() {
+    navigation.navigate('Project');
+  }
+
+  return (
+    <Container>
+      <View style={{ width: '30%' }}>
+        <Photo source={{ uri: `${image}` }} />
+      </View>
+      <ContentContainer>
+        <Name>{name}</Name>
+        <Budget>R$ {budget}</Budget>
+      </ContentContainer>
+      <View style={{ marginTop: 40 }}>
+        <TouchableOpacity onPress={handleNavigateToProject}>
```

Comment:
With this change the opacity of TouchableOpacity is smoother

```suggestion
        <TouchableOpacity onPress={handleNavigateToProject} opacity={0.6}>
```